ZalikovkaC#/Program.cs

```diff
@@ -10,26 +10,26 @@ class Program
     {
         static void Main(string[] args)
         {
-            int k_1 = 6, k_2 = 5, k_3 = 4, k_4 = 4, k_5 = 4, k_6 = 3, k_7 = 2, k_8 = 2;
-            int VM, FZ, INF, HIST, TCA, ECO, ENG, ENG_1, FIZ;
+            const int k_1 = 6, k_2 = 5, k_3 = 4, k_4 = 4, k_5 = 4, k_6 = 3, k_7 = 2, k_8 = 2;
+            int higerMath, fiziks, informaticks, history, TCA, ecology, english, PE;
             Console.WriteLine("Enter the rating for vishmat:");
-            VM = int.Parse(Console.ReadLine());
+            higerMath = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter an assessment in physics:");
-            FZ = int.Parse(Console.ReadLine());
+            fiziks = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter an assessment in computer science:");
-            INF = int.Parse(Console.ReadLine());
+            informaticks = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter an estimate from history:");
-            HIST = int.Parse(Console.ReadLine());
+            history = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter the TCA score:");
             TCA = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter environmental assessment");
-            ECO = int.Parse(Console.ReadLine());
+            ecology = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter an assessment in English:");
-            ENG = int.Parse(Console.ReadLine());
+            english = int.Parse(Console.ReadLine());
             Console.WriteLine("Enter the assessment in physical education:");
-            FIZ = int.Parse(Console.ReadLine());
+            PE = int.Parse(Console.ReadLine());
             double rating;
-            rating = 90.0 * (k_1 * VM + k_2 * FZ + k_3 * INF + k_4 * TCA + k_5 * HIST + k_6 * ECO + k_7 * ENG + k_8 * FIZ)
+            rating = 90.0 * (k_1 * higerMath + k_2 * fiziks + k_3 * informaticks + k_4 * TCA + k_5 * history + k_6 * ecology  + k_7 * english + k_8 * PE)
```

Comment:
Can we divide this formula into parts for instance to calculate the numerator and denominator separately to increase readability?